rfcs/007 - Peer review infra changes.md

```diff
@@ -0,0 +1,125 @@
+# 007 - Peer review infrastructure changes RFC
+
+### Proposed
+
+2022-12-16
```

Comment:
Should the proposal date be changed since this is a revival?

Reply:
sure